tools/c7n_gcp/tests/data/events/k8s_create_pool.json

```diff
@@ -0,0 +1,330 @@
+{
+    "protoPayload": {
+      "@type": "type.googleapis.com/google.cloud.audit.AuditLog",
+      "authenticationInfo": {
+        "principalEmail": "system:node:gke-standard-cluster-1-pool-1-c733eefe-xtbq"
+      },
+      "authorizationInfo": [
+        {
+          "granted": true,
+          "permission": "io.k8s.core.v1.pods.create",
+          "resource": "core/v1/namespaces/kube-system/pods/kube-proxy-gke-standard-cluster-1-pool-1-c733eefe-xtbq"
+        }
+      ],
+      "methodName": "io.k8s.core.v1.pods.create",
+      "request": {
+        "@type": "core.k8s.io/v1.Pod",
+        "apiVersion": "v1",
+        "kind": "Pod",
+        "metadata": {
+          "annotations": {
+            "kubernetes.io/config.hash": "9ef8eaa0a08f35eed9a7f7676a774e8a",
+            "kubernetes.io/config.mirror": "9ef8eaa0a08f35eed9a7f7676a774e8a",
+            "kubernetes.io/config.seen": "2019-05-20T13:23:47.211226997Z",
+            "kubernetes.io/config.source": "file",
+            "scheduler.alpha.kubernetes.io/critical-pod": ""
+          },
+          "creationTimestamp": null,
+          "labels": {
+            "component": "kube-proxy",
+            "tier": "node"
+          },
+          "name": "kube-proxy-gke-standard-cluster-1-pool-1-c733eefe-xtbq",
+          "namespace": "kube-system",
+          "selfLink": "/api/v1/namespaces/kube-system/pods/kube-proxy-gke-standard-cluster-1-pool-1-c733eefe-xtbq",
+          "uid": "9ef8eaa0a08f35eed9a7f7676a774e8a"
+        },
+        "spec": {
```

Comment:
let's remove the spec key in request and response, there's hundreds of lines data here that we don't need.

Reply:
Updated